src/plugins/intel_gpu/src/graph/impls/onednn/concatenation_onednn.hpp

```diff
@@ -54,6 +54,26 @@ struct ConcatenationImplementationManager : public ImplementationManager {
         if (out_layout.data_padding)
             return false;
 
+        auto is_feature_aligned = [](const layout& l) {
```

Comment:
random spot) do we have test case for this behavior? otherwise, could you add one?